Update README.md (#7)
Anon-Planet/PSA: README.md

- [General Rules](#general)
- [Modern Cryprography Room Rules](#moderncrypto)
- [Bnonymity Room Rules](#bnonymity)
- [Serious Offenses](#serious)
- [PSA Bans](#psabans)
- [Ban Appeals](#appeals)

#### General default rules for PSA rooms **unless specific rulesets are mentioned below**:<a name="general"></a>
These are currently enforced on **```#anonymity:matrix.org```**, **```#security:matrix.org```**, **```#OSINT:matrix.org```**, **```#thgtoa-collab:matrix.org```**, and they are not applied on rooms with their own ruleset below.

*We are not lawyers.*  
*We enforce the spirit of the rules, not the letter.*  
*Citing technicalities or imprecisions in the definitions of rules won’t save you.* 

**Avoid drifting too much off-topic or move to off-topic rooms like #psa-offtopic or #bnonymity** 

- Keep it legal
- English only
- Be respectful to each other
- Avoid FUD and/or disinformation
- Avoid gatekeeping and try to remain welcoming to new users
- No hate speech
- If asked, please respect someone's pronouns)
- No spam

> - No doxxing (unless specificaly requested by the person, which you should then take to a DM).

Reviewer: ```suggestion
- No doxxing (unless specifically requested by the person, which you should then take to a DM)
```

- No trolling (this doesn't mean sarcasm is forbidden)
- No NSFW content (no Porn, no Gore, no Hentai)
- No upload of any non-media files (binaries, executables, archives, etc.)
- No voice messages (these will be auto-deleted by the bots)
- Limit political discussions to room topics related issues only.

#### Serious Offenses:<a name="serious"></a>
Some exceptions can apply. Violations will be handled at the discretion of the acting moderator or admin.

> - Zero tolerance for discussion inciting illicit, unethical, hate acts. It is however permitted to discuss these acts as part of a discussion but without actively engaging in inciting or committing those.

Reviewer: ```suggestion
- Zero tolerance for inciting illicit, immoral or unethical acts. It is however permitted to discuss these without actively engaging in inciting or committing the acts. This is limited to the room's topic.
```

  - *Talks about torrenting anonymously are allowed unless the purpose is blatantly illegal.*
- Any CSAM type offense will result in a permanent ban without a possibility of appeal.
  
#### Special Rules for Anonymity room:<a name="anon"></a>

> - **Any** discussion about AnonyPla/Lena's whereabouts is not tolerated but will be tolerated, to an extent, in the ```#Bnonymity:matrix.org``` room.

Reviewer: ```suggestion
- **Any** discussion about AnonyPla/Lena's whereabouts is not tolerated in any capacity except in ```#bnonymity:matrix.org```.
```

-  Talks about Sci-Hub and/or LibGen, Z-lib, etc. are allowed in this room.

#### Rules for Modern Cryptography (```#moderncrypto:gnuradio.org```):<a name="moderncrypto"></a>
See <https://anonymousplanet.org/moderncrypto-rules.html>

#### Rules for Bnonymity (```#bnonymity:matrix.org```):<a name="bnonymity"></a>
- Keep it legal (seriously)
- English only (no Russian, no Chinese, English only)

> - Be "somewhat" respectful to each other but this is "optional"

Reviewer: ```suggestion
- Be kind, don't be a problem
```

Reviewer: > I disagree, shitposting and some level of trolling and slurs have to be accepted in that room as long as those are not hate speech. Like saying "F*** You" is ok to me in Bnon ...

I was more referring to "f-ck this <*entire group of humans*>" which goes hand-in-hand, e.g. with xenophobia, transphobia, etc.

I don't care about a f-bomb here and there, but if it's directed at entire group, it's not acceptable imo. The groups most people are going to come into contact with hate speech are transgender men and women, so I used that as an example of the most common types of abuse one might encounter in modern chat circles. We need to be as inclusive as possible.

- Modern cryptography: ```#modernCrypto:gnuradio.org```
- Open-source intelligence: ```#osint:matrix.org```
- Privacy (Nothing to Hide): ```#privacy:matrix.org```
- THGTOA collaboration discussion: ```#thgtoa-collab:matrix.org```
- Security: ```#security:matrix.org```

This means that those PSA bans are effectively applied on all those rooms and can be issued by PSA Admins and PSA mods (not local mods). See the next section for information about appeals.

#### Ban Appeals:<a name="appeals"></a>
- **Please contact the mods or admins of the room in question to state your case for appealing.**
- You may also contact the admins using the following e-mail: appeals@anonymousplanet.org

Everyone has a right to appeal a ban once (and only once) no matter the ban reason. If the appeal is denied, the user will remain banned without the possibility of a new appeal. If the appeal is granted, the user can be unbanned: 

>   - Probation: this means any violation (even small ones) will void the appeal and end-up un a permanent ban without the possibility of a new appeal. 

Reviewer: ```suggestion
  - Probation: this means any violation (even small ones) will void the appeal and end in a permanent ban (across PSA) without the possibility of a new appeal. 
```

  - Full grant: the appeal is granted without conditions.

#### Abuse:<a name="abuse"></a>

> **This is ONLY for reporting moderation abuse. Do not use this contact lightly and without proofs**.

Reviewer: ```suggestion
**This is ONLY for reporting moderation abuse. Do not use this contact lightly or without proof**. Mods are expected to act in a capacity that is both fair and respectful. Abuse is taken very seriously. Please report all violations by mods.
```

- **Please contact the other mods or admins of the room in question to report moderation abuse.**

> - You may also contact the admins using the following e-mail: abuse@anonymousplanet.org (warning, this e-mail might be received by the accused, but as it is the case for any laws/rules. **Everyone has the right to face their accuser¨**. 

Reviewer: ```suggestion
- You may also contact the admins using the following e-mail: abuse@anonymousplanet.org (warning, this e-mail might be received by the accused, but as it is the case for any laws/rules. **"Everyone has the right to face their accuser"**. 
```
